test/Microsoft.ML.Tests/Scenarios/TensorflowTests.cs

```diff
@@ -55,17 +55,17 @@ public void TensorFlowTransforCifarEndToEndTest()
             {
                 ImagePath = GetDataPath("images/banana.jpg")
             });
-            Assert.Equal(1, prediction.PredictedScores[0], 2);
-            Assert.Equal(0, prediction.PredictedScores[1], 2);
+            Assert.Equal(0, prediction.PredictedScores[0], 2);
+            Assert.Equal(1, prediction.PredictedScores[1], 2);
             Assert.Equal(0, prediction.PredictedScores[2], 2);
 
             prediction = predictFunction.Predict(new CifarData()
             {
                 ImagePath = GetDataPath("images/hotdog.jpg")
             });
             Assert.Equal(0, prediction.PredictedScores[0], 2);
-            Assert.Equal(1, prediction.PredictedScores[1], 2);
-            Assert.Equal(0, prediction.PredictedScores[2], 2);
+            Assert.Equal(0, prediction.PredictedScores[1], 2);
+            Assert.Equal(1, prediction.PredictedScores[2], 2);
```

Comment:
Might be a good idea  to verify the  PredictedScores on for  "tomato.bmp" (also "tomato.jpg")